pkg/ingester/ingester_test.go

```diff
@@ -4434,29 +4430,8 @@ func TestIngester_inflightPushRequests(t *testing.T) {
 
 	g, ctx := errgroup.WithContext(ctx)
 	g.Go(func() error {
-		count := 100000
-		target := time.Second
-
-		// find right count to make sure that push takes given target duration.
-		for {
-			req := generateSamplesForLabel(labels.FromStrings(labels.MetricName, fmt.Sprintf("test-%d", count)), count)
-
-			start := time.Now()
-			_, err := i.Push(ctx, req)
-			require.NoError(t, err)
-
-			elapsed := time.Since(start)
-			t.Log(count, elapsed)
-			if elapsed > time.Second {
-				break
-			}
-
-			count = int(float64(count) * float64(target/elapsed) * 1.5) // Adjust number of samples to hit our target push duration.
-		}
-
-		// Now repeat push with number of samples calibrated to our target.
```

Comment:
is this change related to this PR?

Reply:
It's not, I can move it into a separate PR if you'd like. The change was included to address an issue with the test's flakiness. I added a note about it in the description of the PR:

> The TestIngester_inflightPushRequests test was updated as well since it could fail if there were too many samples pushed (3600000 (also number of seconds in an hour)) and an out of bounds error would be thrown.

Reply:
Its ok!